LottieDropper/Dropbox/DropboxDetailViewModel.h

```diff
@@ -7,6 +7,7 @@
 //
 
 #import <Foundation/Foundation.h>
+#import "Connectivity.h"
```

Comment:
Een detail moet geen kennis hebben van connectivity. Dat is de taak van de lijst. De taak van een detail is gewoon een file laden. Hij moet dus twee functies hebben 

1. Laden van lokale file
2. File refreshen/initieel laden van vanaf dropbox.